# goal
---

# result
bool ok
---

# feedback
float32 percent_complete